src/ophys_etl/modules/maximum_projection/utils.py

```diff
@@ -0,0 +1,294 @@
+import pathlib
```

Comment:
These seem like more than utils to me. What are your thoughts on creating a `MaximumProjection` class, and adding a `@classmethod` `from_path` to instantiate it?

Reply:
Is this more than an aesthetic choice?

From my perspective, the output is just a numpy array, so we don't need to define a special output class, and there isn't a state that needs to be carried around during processing. I see the simplicity in having `FilteredMaximumProjection.from_array()` and `Filtered MaximumProjection.from_path()`. It seems overly complicated to me to have those methods return a numpy array, rather than some more complicated object.

Reply:
If not a class, then I would move the logic out of `utils` and make the functions private which are local to the module